stackslib/src/net/api/mod.rs

```diff
@@ -114,6 +115,7 @@ impl StacksHttp {
         self.register_rpc_endpoint(postmicroblock::RPCPostMicroblockRequestHandler::new());
         self.register_rpc_endpoint(poststackerdbchunk::RPCPostStackerDBChunkRequestHandler::new());
         self.register_rpc_endpoint(posttransaction::RPCPostTransactionRequestHandler::new());
+        self.register_rpc_endpoint(getstackers::GetStackersRequestHandler::default());
```

Comment:
Miner code aesthetic: can you place the call to `self.register_rpc_endpoint()` in alphabetical order with the other RPC endpoints?  So, after the one for `getstackerdbmetadata`?

Reply:
Sure, but are we sure it shouldn't be after `getstackerdbchunk` and before `getpoxinfo` ;) ?